### DISCUSSION TITLE
REDACTED

### DISCUSSION THREAD

**Discussion by Futuraura** [community/community#128539] [GitHub Education]
CENSORED FOR PRIVACY REASONS!

**Comment by davevad93** [comment]
Hi @Futuraura ,

If you don't have a school ID you should try with some other school issued document which proves your current student status. You can also try uploading a screenshot of your most recent exam results, that's what I did and it worked for me. Most importantly, it has to be a document that shows your name, the name of the educational institution and a current date.

**Reply by davevad93** [reply]
It should be enough but it has to display a current date in a `YYYY/MM/DD` or `DD/MM/YYYY` format, otherwise it wouldn't be enough for GitHub as proof of your current enrollment. As far as I know the application page only permits uploading `PNG` or `JPEG` files and I'm not aware if it's possible to make the request "manually" so you could either wait till August or try contacting support by opening a ticket [here](https://support.github.com/contact/education) and explain your situation.

**Comment by samus-aran** [comment]
Hi @Futuraura 

We don't accept PDF and we do prefer to use live capture i.e. photos. Are you able to take a photo of the PDF (or print it out)? We do allow users to upload documents whilst on campus if you wish to wait until you are attending to complete the application.

We've also updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. 👍
